resources/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from .models import Resource
+
+
+class ResourceForm(forms.ModelForm):
+    """Allows admin to add/edit resources"""
+
+    class Meta:
+        model = Resource
+        fields = '__all__'
+
+
```

Comment:
As per PEP8 method definitions inside a class are surrounded by a single blank line. Can you please remove the extra blank line?

Reply:
Sure, think I've done this